src/firestore/src/status.rs

```diff
@@ -174,12 +174,10 @@ mod test {
                     description: "desc".into(),
                     ..Default::default()
                 }],
-                ..Default::default()
```

Comment:
Hilariously, removing these sets us up for test breaks if/when new fields are added to the protos.

Aside: seems like the prost messages should be marked `#[non_exhaustive]`.